### PR TITLE
Block Iptables on all the nodes where pods are present

### PR DESCRIPTION
**Pull request by vprabhakar-px**
Block Iptables on all the nodes where pods are present

### CODE REVIEW

File: tests/basic/customer_issues_test.go [hunk 1]
```diff
@@ -1077,7 +1077,6 @@ var _ = Describe("{FADAPodRecoveryAfterBounce}", func() {
 	var k8sCore = core.Instance()
 
 	itLog := "FADAPodRecoveryAfterBounce"
-	stepLog := "App Stuck in ContainerCreation State with Device Exists in the backend"
 	It(itLog, func() {
 		log.Infof(stepLog)
```
Comment (lsrinivas-pure) [inline tests/basic/customer_issues_test.go:1081]: I think you need to delete this also if stepLog deletion is intentional

File: tests/basic/customer_issues_test.go [hunk 2]
```diff
@@ -1204,3 +1203,182 @@ var _ = Describe("{FADAPodRecoveryAfterBounce}", func() {
...
+			Expectation : PODS should be up and running after connection restored.
+
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("FADAPodRecoveryAllPathDownUsingIptableRule",
+			"Verify Pod Recovers from RO mode after Bounce after blocking iptable Rules",
+			nil, 0)
+	})
+	var contexts []*scheduler.Context
+	var k8sCore = core.Instance()
```
Comment (lsrinivas-pure) [inline tests/basic/customer_issues_test.go:1224]: move it inside it

File: tests/basic/customer_issues_test.go [hunk 3]
```diff
@@ -1204,3 +1203,182 @@ var _ = Describe("{FADAPodRecoveryAfterBounce}", func() {
...
+				log.Infof("Validating test on Pod [%v]", eachPod)
+				podNode, err := GetNodeFromIPAddress(eachPod.Status.HostIP)
+				log.FailOnError(err, "unable to find the node from the pod")
+				// Revert iscsi rules that was set on the node
+				err = blockiSCSIPortOnNode(podNode, false)
+			}
+		})
+
+		// Sleep for some time for Px to come up online and working
+		time.Sleep(10 * time.Minute)
```
Comment (lsrinivas-pure) [inline tests/basic/customer_issues_test.go:1339]: can we use waitfordriveruponnode here

File: tests/basic/customer_issues_test.go (hunk 2 above)
Comment (vprabhakar-px) [reply tests/basic/customer_issues_test.go]: done

File: tests/basic/customer_issues_test.go (hunk 1 above)
Comment (vprabhakar-px) [reply tests/basic/customer_issues_test.go]: done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**